server/src/yml-objects/YmlObjectInstance.ts

```diff
@@ -8,7 +8,14 @@ export class YmlObjectInstance extends AbstractYmlObject {
     public domainsLevel2: YmlType;
     public readonly kind = CompletionItemKind.Variable;
 
-    constructor(indentifier: string, uri: string) {
+    constructor(indentifier: string, uri: string, public readonly isLocal) {
```

Comment:
What is that public readonly thingy? xD (I mean, those keywords)

Reply:
`public readonly` in a constructor allows to define and set the value of a final attribute for the corresponding class.
So, here as say that my class now has an `isLocal` final attribute and when the constructor is called, we set its value directly.